src/System.Net.WebSockets.WebSocketProtocol/src/System.Net.WebSockets.WebSocketProtocol.csproj

```diff
@@ -16,7 +16,7 @@
     <Compile Remove="System\Net\WebSockets\ManagedWebSocketExtensions.netstandard.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == '$(NetFrameworkMinimum)' or '$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
```

Comment:
Can you please explain this change? I was not aware that TFI would also

Reply:
I'm using a negation here. Meaning anything that is not NETCoreApp which includes NETStandard.